client/gettext/po/manageiq-ui-service.pot

```diff
@@ -4,7 +4,7 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Project-Id-Version: \n"
 
-#: ./client/app/states/dashboard/dashboard.html:88
+#: ./client/app/components/dashboard/dashboard.html:88
```

Comment:
recreating `pot` files should not be a part of the PR

Reply:
@himdel will remove, but why not? especially when file refactoring takes place?